src/entities/viewer/model/api.ts

```diff
@@ -0,0 +1,11 @@
+import { createEvent, createStore } from 'effector';
+import { Auth0Client } from '@auth0/auth0-spa-js';
+import { authConfig } from '../../../shared';
+
+export const auth0 = new Auth0Client(authConfig);
+
+export const setUser = createEvent();
```

Comment:
check effector's naming convention for events

Reply:
fixed